src/visitor/mysql.rs

```diff
@@ -441,19 +441,38 @@ impl<'a> Visitor<'a> for Mysql<'a> {
         Ok(())
     }
 
-    #[cfg(feature = "postgresql")]
-    fn visit_text_search(&mut self, _text_search: crate::prelude::TextSearch<'a>) -> visitor::Result {
-        unimplemented!("Full-text search is not yet supported on MySQL")
-    }
-
-    #[cfg(feature = "postgresql")]
-    fn visit_matches(
-        &mut self,
-        _left: Expression<'a>,
-        _right: std::borrow::Cow<'a, str>,
-        _not: bool,
-    ) -> visitor::Result {
-        unimplemented!("Full-text search is not yet supported on MySQL")
+    #[cfg(any(feature = "postgresql", feature = "mysql"))]
```

Comment:
It's weird that we need this for the crate to compile, but I think I understand why.

Reply:
I'm open to better ways of doing that

Reply:
IIUC, we don't need those for the QE. We need those because Quaint is open-source and we want to offer fine-grained APIs based on which features you've enabled. QE side we always enable all of them anyway since we don't compile different QEs for different connectors.
This might turn out to be useful in the future though.

Reply:
yep, that's the reasoning. I wonder if there would be ways to have these feature gates in fewer places.

Reply:
maybe group methods in modules and have feature gates on those

Reply:
Definitely a larger project that doesn't fit in this PR though